events/README.md

```diff
@@ -0,0 +1,416 @@
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```json
+      {
+        "message": "session is ending"
+      }
+      ```
+- **Error Response:**
+  - **Code:** 401
+    - **Content:**
+      **`{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`**
+  - **Code:** 500
+    - **Content**
+      **`{ 'statusCode': 500, 'error': 'Internal server error', 'message': 'Couldn't end the room. Please try again later'}`**
+
+## GET - /session
+
+To retrieve all the sessions.
+
+- Params:
+  - hits: Number of sessions in one response
+  - offset: Start of response
+    - ex: For 1-10 hits: 10, offset: 0; 11-20 hits: 10, offset: 10;
+- Query
+  - None
+- Body
+  - None
+- Headers
+  - Content-Type: application/json
+  - Authorization: Bearer <management_token>
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```jsx
+      {
+      	"limit": 10,
+      	"data": [
+      		<SESSION_OBJECT>,
+      		<SESSION_OBJECT>,
+      	]
+      }
+      ```
+- **Error Response:**
+  - **Code:** 401
+    - **Content:**
+      `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
+  - **Code:** 500
+    - **Content**
+      `{ 'statusCode': 500, 'error': 'Internal server error', 'message': 'Couldn't get sessions. Please try again later'`
+
+## GET - /session/:status
+
+To get currently running sessions.
+
+- Params:
+  - active: true/false
+  - hits: Number of sessions in one response
+  - offset: Start of response
+    - ex: For 1-10 hits: 10, offset: 0; 11-20 hits: 10, offset: 10;
+- Query
+  - None
+- Body
+  - None
+- Headers
+  - Content-Type: application/json
+  - Authorization: Bearer <management_token>
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```jsx
+      {
+      	"limit": 10,
+      	"data": [
+      		<SESSION_OBJECT>,
+      		<SESSION_OBJECT>,
+      	]
+      }
+      ```
+- **Error Response:**
+  - **Code:** 401
+    - **Content:**
+      `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
+  - **Code:** 500
+    - **Content**
+      `{ 'statusCode': 500, 'error': 'Internal server error', 'message': 'Couldn't get sessions. Please try again later'`
+
+## GET - /session/:id=<SESSION_ID>
+
+Retrieves the details of a specific active session.
+
+- **Params:**
+  - **`id=[string]`** (The ID of the session to retrieve)
+- **Query**
+  - None
+- **Body**
+  - None
+- **Headers**
+  - Authorization: Bearer <management_token>
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```json
+      {
+        "id": "<session_id>",
+        "room_id": "<room_id>",
+        "customer_id": "<customer_id>",
+        "active": false,
+        "peers": {
+          "peer_id": {},
+          "peer_id": {}
```

Comment:
`peer_id` in the beginning were strings. can you crosscheck once.

Reply:
Yaa, these would be objects of peer here. Since, it's a response. I have wrote it in a better format now.